PowerEditor/src/WinControls/FunctionList/functionListPanel.cpp

```diff
@@ -446,14 +446,14 @@ void FunctionListPanel::findMarkEntry(HTREEITEM htItem, LONG line)
 					{
 						if (sci_line > _findLine || _findLine == -1)
 						{
-							_findLine = sci_line;
+							_findLine = static_cast<LONG>(sci_line);
 							_findItem = htItem;
 						}
 					}
 					else
 					{
 						if (sci_line < _findEndLine)
-							_findEndLine = sci_line;
+							_findEndLine = static_cast<LONG>(sci_line);

```

Comment:
Why not change
`auto sci_line = (*_ppEditView)->execute(SCI_LINEFROMPOSITION, pos);`
into
`LONG sci_line = static_cast<LONG>((*_ppEditView)->execute(SCI_LINEFROMPOSITION, pos));`
so you only have to cast once?

Reply:
Agreed. Implemented. Thanks!